tests/acceptance/programs-test.js

```diff
@@ -156,6 +156,7 @@ test('remove program', function(assert) {
     });
   });
   andThen(function(){
+    assert.equal(getElementText(find('.flash-messages')),getText(''));
```

Comment:
Is this test asserting that there is no flash message content?  If so I think the right test would be 
```javascript
assert.equal(find('.flash-messages').length, 0);
```